src/components/ThemeToggle/ThemeToggle.stories.tsx

```diff
@@ -58,24 +41,7 @@ export default {
 
 type Story = StoryObj<typeof ThemeToggle>
 
-// We over ride the normal option of multiple themes in happo
-// since we want to only show this specific one. Dynamically
-// switching the theme in storybook doesn't work well with
-// the theme toggle button
-export const WithUserLight: Story = {
-  parameters: {
-    happo: {
-      themes: ['light'],
-    },
-  },
-}
-export const WithUserDark: Story = {
-  parameters: {
-    happo: {
-      themes: ['dark'],
-    },
-  },
-}
```

Comment:
I'm conflicted on the storybook changes. They were done to support the ThemeToggle since how it retrieved user data was changed. However, since reverting the changes they are not needed. 

Conflict: Should I leave two stories? Leave the original one? Keep some more of the changes? I am inclined to remove them but could be talked into leaving them.